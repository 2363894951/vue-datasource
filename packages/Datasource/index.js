import Datasource from './src/main.vue';

Datasource.install = (Vue) => {
  Vue.component(Datasource.name, Datasource);
};

export default Datasource;
