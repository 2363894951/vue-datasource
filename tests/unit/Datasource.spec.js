import { shallowMount } from '@vue/test-utils';
import Datasource from '../../packages/Datasource/src/main.vue';

const vm = shallowMount(Datasource);

describe('Datasource.vue', () => {
  it('should match the snapshot', () => {
    expect(vm.$el).toMatchSnapshot();
  });
});
