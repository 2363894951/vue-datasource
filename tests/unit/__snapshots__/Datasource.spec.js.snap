// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Datasource.vue should match the snapshot 1`] = `undefined`;
